src/main.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable max-classes-per-file */
```

Comment:
> Files containing multiple classes can often result in a less navigable and poorly structured codebase. Best practice is to keep each file limited to a single responsibility.

https://eslint.org/docs/rules/max-classes-per-file

Consider separation in the future in the future

Reply:
@probil, I have a PR for this